README.md

```diff
@@ -43,6 +43,7 @@ require('vesper').setup({
         variables = true, -- Boolean: Italicizes variables
     },
     overrides = {}, -- A dictionary of group names, can be a function returning a dictionary or a table.
+  palette_overrides = {}
```

Comment:
Please keep the indentation of the other lines here. Probably adding two spaces to the left is enough. Like this:

```md
    palette_overrides = {}
```